notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1118_ims_slewing_azimuth_analysis_positioning.ipynb

```diff
@@ -0,0 +1,1011 @@
+{
```

Comment:
<p>In the future, considering creating a module inside <code>python/lsst/sitcom/vandv/m1m3</code> with the name of the technote (see <a href="see [sitcomtn092.py](https://github.com/lsst-sitcom/notebooks_vandv/blob/develop/python/lsst/sitcom/vandv/m1m3/sitcomtn092.py" target="_blank">sitcomtn092.py</a> or <a href="see sitcomtn092.py or [sitcom1089.py](https://github.com/lsst-sitcom/notebooks_vandv/blob/develop/python/lsst/sitcom/vandv/m1m3/sitcom1089.py" target="_blank">sitcom1089.py</a> as examples). You can move large cells like this to that module as functions or classes and the notebook gets much cleaner.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I understand the rationale, but for this kind of simple notebook I prefer to have everything at the same place. I think that it makes things simpler for someone else to reuse and adapt the notebook. </p>